scripts/install/opt/seagate/cortx/motr/bin/motr_setup

```diff
@@ -179,6 +179,7 @@ class ResetCmd(Cmd):
         super().__init__(args)
 
     def process(self):
+        remove_data(self)
```

Comment:
can we have this as a name remove_logs instead of remove_date

Reply:
3c8ee61